contributing_to_docs/doc_guidelines.adoc

```diff
@@ -1815,6 +1879,16 @@ Only use `.` formatting (`.Additional resources`) in a module or to follow a mod
 * xref:../configuration/product-settings.adoc#installation-parameters_product-settings[Product installation configuration parameters]
 ----
 
+You can add link, hyperlinks, and cross references in either the _Additional resources_ section or the _Next steps_ section, but know the following differences:
+
+Additional resources - module level:: Requires the `.Additional resources` section heading. The listed resources link to other material closely related to the contents of the module. Focus on including the most relevant resources for the user, not a full list of possibly related resources.
+
+Additional resources - assembly level:: Requires the `== Additional resources` section heading. If a resource applies to all of the modules in a user story, consider listing the resource in the _Additional resources_ section of the assembly file.
```

Comment:
Minor suggestion just to drill this home:
```suggestion
Additional resources - assembly level:: Requires the `== Additional resources` section heading with a unique anchor ID. If a resource applies to all of the modules in a user story, consider listing the resource in the _Additional resources_ section of the assembly file.
```

Reply:
Yeah, good point. I found a vSphere ID in Nutanix docs recently and it was driving builds crazy. A poor copy and paste operation.